### PR TITLE
HomeScreen 'isClickable' fix

### PR DESCRIPTION
**Pull request by Jurgenvs**
HomeScreen better reflects if the stroll button is clickable. It is not perfect, but before fixing more I think we should see to it that event screens can't appear when going back to the main menu. Menu works correct when no stroll is running.

### CODE REVIEW

File: core/src/cg/group4/view/screen/SettingsScreen.java [hunk 1]
```diff
@@ -88,7 +109,22 @@ protected ChangeListener resetStrollBehaviour() {
         return new ChangeListener() {
             @Override
             public void changed(final ChangeEvent event, final Actor actor) {
-                TimeKeeper.getInstance().getTimer(Timer.Global.STROLL.name()).reset();
+                cStrollTimer.reset();
+                //cScreenStore.setScreen(new RewardScreen(cScreenStore));
```
Comment (sirpolomotro) [inline core/src/cg/group4/view/screen/SettingsScreen.java]: Commented code

File: core/src/cg/group4/view/screen/SettingsScreen.java [hunk 2]
```diff
@@ -88,7 +109,22 @@ protected ChangeListener resetStrollBehaviour() {
...
+     * Stops the stroll timer.
+     *
+     * @return ChangeListener
+     */
+    protected ChangeListener stopStrollBehaviour() {
+        return new ChangeListener() {
+            @Override
+            public void changed(final ChangeEvent event, final Actor actor) {
+                cStrollTimer.stop();
                 //cScreenStore.setScreen(new RewardScreen(cScreenStore));
```
Comment (sirpolomotro) [inline core/src/cg/group4/view/screen/SettingsScreen.java]: Commented code

**Comment by sirpolomotro** [comment]
Issue should be made of progressive stroll... not sure what it should do though.

**Comment by foresterre** [comment]
Comments are now removed. The other parts of look fine.

**Pull request merged**